src/CONST.js

```diff
@@ -1036,264 +1036,1081 @@ const CONST = {
         SC: 'Seychelles',
-        SD: 'Sudan',
-        SE: 'Sweden',
+        SL: 'Sierra Leone',
         SG: 'Singapore',
-        SH: 'St. Helena',
-        SI: 'Slovenia',
-        SJ: 'Svalbard & Jan Mayen',
+        SX: 'Sint Maarten',
         SK: 'Slovakia',
-        SL: 'Sierra Leone',
-        SM: 'San Marino',
-        SN: 'Senegal',
+        SI: 'Slovenia',
+        SB: 'Solomon Islands',
         SO: 'Somalia',
-        SR: 'Suriname',
+        ZA: 'South Africa',
+        GS: 'South Georgia & South Sandwich Islands',
+        KR: 'South Korea',
         SS: 'South Sudan',
-        ST: 'São Tomé & Príncipe',
-        SV: 'El Salvador',
-        SX: 'Sint Maarten',
-        SY: 'Syria',
+        ES: 'Spain',
+        LK: 'Sri Lanka',
+        SH: 'St. Helena',
+        KN: 'St. Kitts & Nevis',
+        LC: 'St. Lucia',
+        MF: 'St. Martin',
+        PM: 'St. Pierre & Miquelon',
+        VC: 'St. Vincent & Grenadines',
+        SD: 'Sudan',
+        SR: 'Suriname',
+        SJ: 'Svalbard & Jan Mayen',
         SZ: 'Swaziland',
-        TA: 'Tristan da Cunha',
-        TC: 'Turks & Caicos Islands',
-        TD: 'Chad',
-        TF: 'French Southern Territories',
-        TG: 'Togo',
-        TH: 'Thailand',
+        SE: 'Sweden',
+        CH: 'Switzerland',
+        SY: 'Syria',
+        TW: 'Taiwan',
         TJ: 'Tajikistan',
-        TK: 'Tokelau',
+        TZ: 'Tanzania',
+        TH: 'Thailand',
         TL: 'Timor-Leste',
-        TM: 'Turkmenistan',
-        TN: 'Tunisia',
+        TG: 'Togo',
+        TK: 'Tokelau',
         TO: 'Tonga',
-        TR: 'Turkey',
         TT: 'Trinidad & Tobago',
+        TA: 'Tristan da Cunha',
+        TN: 'Tunisia',
+        TR: 'Turkey',
+        TM: 'Turkmenistan',
+        TC: 'Turks & Caicos Islands',
         TV: 'Tuvalu',
-        TW: 'Taiwan',
-        TZ: 'Tanzania',
-        UA: 'Ukraine',
-        UG: 'Uganda',
         UM: 'U.S. Outlying Islands',
-        UN: 'United Nations',
+        VI: 'U.S. Virgin Islands',
+        UG: 'Uganda',
+        UA: 'Ukraine',
+        AE: 'United Arab Emirates',
+        GB: 'United Kingdom',
         US: 'United States',
         UY: 'Uruguay',
         UZ: 'Uzbekistan',
+        VU: 'Vanuatu',
         VA: 'Vatican City',
-        VC: 'St. Vincent & Grenadines',
         VE: 'Venezuela',
-        VG: 'British Virgin Islands',
-        VI: 'U.S. Virgin Islands',
         VN: 'Vietnam',
-        VU: 'Vanuatu',
         WF: 'Wallis & Futuna',
-        WS: 'Samoa',
-        XK: 'Kosovo',
+        EH: 'Western Sahara',
         YE: 'Yemen',
-        YT: 'Mayotte',
-        ZA: 'South Africa',
         ZM: 'Zambia',
         ZW: 'Zimbabwe',
     },
 
+    // Sources: https://github.com/Expensify/App/issues/14958#issuecomment-1442138427
+    // https://github.com/Expensify/App/issues/14958#issuecomment-1456026810
+    COUNTRY_ZIP_VALIDATION_REGEX: {
+        AC: {
+            zipRegex: /^ASCN 1ZZ$/,
+            zipFormat: 'ASCN 1ZZ',
+        },
```

Comment:
What do y'all think about modifying this to something like:

```suggestion
    COUNTRY_ZIP_REGEX_DATA: {
        AC: {
            regex: /^ASCN 1ZZ$/,
            samples: 'ASCN 1ZZ',
        },
```

I think we can get rid of `zip` in the inner properties of each object, since we already know this is all zip code regex information from the main title `COUNTRY_ZIP_REGEX_DATA`.  Also, "format" and "regex" seem too similar, I think `samples` makes a bit more sense there